manifests/crd.yaml

```diff
@@ -6,43 +6,35 @@ metadata:
   annotations:
     controller-gen.kubebuilder.io/version: v0.6.2
   creationTimestamp: null
-  name: backups.pingcap.com
+  name: backupschedules.pingcap.com
```

Comment:
Is this modification expected? @liubog2008 

Seems revert the change of https://github.com/pingcap/tidb-operator/pull/4530/files#diff-c2679bcbd5085af02dbd20e3140549fdea75faaf8c6d27746d2e61c53adb9fa4.

I was expecting only add `scalePolicy` in this file but actually not. That's what I mean "too much". And I am not quite sure it's right or not.

Reply:
@lizhemingi It should be OK. It seems just the order issue of the two crds.